File: publish/scripts/tools/create_cpp_project/proj/clean.py
import os
import shutil

def removeFile(f):
    if os.path.isfile(f):
        os.remove(f)

removeFile("CppApplicationName")
removeFile("CppApplicationName.exe")
removeFile("Makefile")
removeFile("CMakeCache.txt")
removeFile("cmake_install.cmake")
shutil.rmtree("CMakeFiles", True)
shutil.rmtree("Debug", True)
shutil.rmtree("Release", True)
shutil.rmtree("debug", True)
shutil.rmtree("release", True)

print("Clean ok.")

File: publish/scripts/tools/create_cpp_project/proj/clean.lua
util.pathRemove('CppApplicationName')
util.pathRemove('CppApplicationName.exe')
util.pathRemove('Makefile')
util.pathRemove('CMakeCache.txt')
util.pathRemove('cmake_install.cmake')
util.pathRemoveAll('CMakeFiles')
util.pathRemoveAll('Debug')
util.pathRemoveAll('Release')
util.pathRemoveAll('debug')
util.pathRemoveAll('release')

print('Clean ok.')

